website/blog/2023-02-28-ferretdb-release-0-9-2.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "FerretDB v0.9.2 - Tigris Authentication support"
+slug: ferretdb-v-0-9-2
+authors: [alex]
+description: "We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more."
+image: /img/blog/ferretdb_v0.9.2.jpg
+tags: [release]
+date: 2023-02-28
+---
+
+We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more.
+
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb_v0.9.2.jpg)
+
+<!--truncate-->
+
+It’s been an intense and thrilling process toward bringing you the open-source alternative to FerretDB.
```

Comment:
The past few weeks were exciting: our cooperation with the open source community is stronger than ever. The feedback of developers and service providers makes us confident that we are on the right track with creating  truly open source alternative to MongoDB.